relay_rpc/src/jwt.rs

```diff
@@ -149,12 +149,17 @@ pub trait VerifyableClaims: Serialize + DeserializeOwned {
 
         // Decode header.
         data_encoding::BASE64URL_NOPAD
-            .decode_mut(header.as_bytes(), &mut output[..header_len])
+            .decode_mut(
```

Comment:
I think these are all unsafe slices already, but replacing with error catching instead of clippy allow because we are already using Results and there seems to be 1 shared Error variant for all cases here.